bin/docker/run

```diff
@@ -5,12 +5,36 @@
 # http://www.boost.org/LICENSE_1_0.txt)
 set -e
 
+# CLI handling
+
+pass_credentials_bintray=
+cmd=""
+
+while [ "$1" != "" ]; do
+    param="$1"
+
+    case $param in
+        --credentials-bintray)
+            pass_credentials_bintray=true
+            ;;
+        *)
+            cmd="$@"
+            break
+            ;;
+    esac
+
+    shift
+done
```

Comment:
I think this should be done by `beaver bintray upload` actually, not the general `run` command, and this can be easily achieved by injecting variables to pass using `BEAVER_DOCKER_VARS` (or whatever it was called :D).